cinema/views.py

```diff
@@ -1,10 +1,15 @@
 from datetime import datetime
 
 from django.db.models import F, Count
-from rest_framework import viewsets
+from rest_framework import viewsets, status
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.decorators import permission_classes
```

Comment:
The `permission_classes` decorator is imported but not used in the file. If you intend to use it, make sure to apply it to the appropriate view functions or remove the import if it's unnecessary.